apps/rush-lib/src/api/test/RushConfiguration.test.ts

```diff
@@ -118,6 +118,86 @@ describe('RushConfiguration', () => {
     done();
   });
 
+  it('can load repo/rush-npm-extended-project.json', (done: jest.DoneCallback) => {
+    const rushFilename: string = path.resolve(__dirname, 'repo', 'rush-npm-extended-projects.json');
+
+    const rushConfiguration: RushConfiguration = RushConfiguration.loadFromConfigurationFile(rushFilename);
+
+    rushConfiguration.projectsExtended = [
```

Comment:
These modifications should be in the JSON file directly.

Reply:
Agreed and changed